projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -244,9 +253,23 @@ export class UpdateChanges {
                     fileContent = this.copyPropertyValueBetweenElementTags(fileContent, match, moveMatch);
                 }
 
+                if (change.valueTransform) {
+                    // We need to remove 'g' flag to see capture groups
+                    // This will FAIL if there are multiple bindings in A SINGLE TAG to the SAME property
```

Comment:
It actually won't since the actual replace still uses the `reg` that is global. Bet you can put it in a test to make sure :)